macosx-aarch64/Makefile

```diff
@@ -0,0 +1,18 @@
+CC = cc
+SRC = alpn.c clientcert.c options.c session.c ssl.c threads.c util.c
+OBJ = $(patsubst %.c, target/%.o, $(SRC))
+ARCH = arm64
+
+default: target/classes/macosx-aarch64/libwfssl.dylib
+
+clean:
+	rm -rf target
```

Comment:
I see that all other Makefiles do the same, but IMO it should clean just its output, i.e. `target/classes/macosx-aarch64`

Reply:
In this case mvn clean will remove the target folders and their content anyway - yes - a bit of overkill I agree.